WooCommerce/Classes/ViewRelated/Blaze/CampaignCreation/BlazeCampaignCreationFormViewModel.swift

```diff
@@ -92,11 +99,47 @@ final class BlazeCampaignCreationFormViewModel: ObservableObject {
     @Published private(set) var targetLanguageText: String = ""
     @Published private(set) var targetDeviceText: String = ""
 
+    // AI Suggestions
+    @Published private(set) var isLoadingAISuggestions: Bool = true
+    private let storage: StorageManagerType
+    private var product: Product? {
+        guard let product = productsResultsController.fetchedObjects.first else {
+            assertionFailure("Unable to fetch product with ID: \(productID)")
+            return nil
+        }
+        return product
+    }
+
+    @Published private(set) var errorState: ErrorState = .none
+    private var suggestions: [BlazeAISuggestion] = []
+
+    var canConfirmDetails: Bool {
+        guard !isDownloadingImage else {
+            return false
+        }
+        return tagline.isNotEmpty && description.isNotEmpty
+    }
+
+    /// ResultController to to track the current product count.
```

Comment:
small nit: inaccurate comment? ("count")

Reply:
Updated in https://github.com/woocommerce/woocommerce-ios/pull/11658/commits/186ce1cbfa4da2782e8cbeed900260409ae06077